mesos-client/src/main/scala/mesosphere/mesos/conf/MesosClientConf.scala

```diff
@@ -0,0 +1,18 @@
+package mesosphere.mesos.conf
+
+import scala.concurrent.duration._
+
+/**
+  * Configuring mesos v1 client
+  *
+  * @param master The URL of the Mesos master
+  * @param sourceBufferSize Buffer size of the mesos source im messages
+  * @param redirectRetries Number of retries to follow mesos master redirect
+  * @param idleTimeout Time in seconds between two processed elements exceeds the provided timeout then the connection to mesos
+  *                    is interrupted. Is usually set to approx. 5 hear beats.
+  */
+case class MesosClientConf(
```

Comment:
Should we apply the same configuration patter here as in [metrics](https://github.com/mesosphere/usi/pull/13/files#r251202266) e.g. using Typesafe config and this renamed to `MesosClientSettings`?

Reply:
+1